tests/tools/oecert/oecert.edl

```diff
@@ -19,5 +19,11 @@ enclave {
             size_t public_key_size,
             [out] unsigned char **data, 
             [out] size_t *data_size);
+
+        public oe_result_t get_remote_report_with_custom_data(
```

Comment:
the term "report" is legacy, is this really needed or can we use "evidence" instead?